File: model-a-person-in-js/person.js
let person = {
    firstname:"Leslie",
    lastname:"Wilson",
    hometown:"New York",
    occupation:"Student",
    catchPhrase:"tap that a$$",
    name() {
        return`${person.firstname} ann ${person.lastname}`
    },
}
console.log(person)
console.log(person.name())

File: person-constructor/person.rb
class Person
    def initialize(name, last_name = nil)

        if last_name.nil?
            full_name = name.split


        @first_name = full_name[0]

        @last_name = full_name[1]
    else
        @first_name = name
        @last_name = last_name
    end

    end
end

Person.new('leslie','wilson')
